stories/components/todo/todo.stories.js

```diff
@@ -25,6 +25,14 @@ export const tasks = () => html`
   <mgt-todo></mgt-todo>
 `;
 
+export const tasksWithTargetId = () => html`
+  <mgt-todo target-id="AQMkADhiZTI2MGUyLTZjYmUtNDNlYS05YTJkLWNmNzA4MTIwNWI5YQAuAAAD7zBnIbVuBE6ECinrQyLxwwEAcUAkVjKRgUeDolDvaYAd0AAAAgESAAAA"></mgt-todo>
+`;
+
+export const tasksWithInitialId = () => html`
+  <mgt-todo initial-id="AQMkADhiZTI2MGUyLTZjYmUtNDNlYS05YTJkLWNmNzA4MTIwNWI5YQAuAAAD7zBnIbVuBE6ECinrQyLxwwEAcUAkVjKRgUeDolDvaYAd0AAAAgESAAAA"></mgt-todo>
```

Comment:
Let's use the default task list in the mock provider here

```suggestion
  <mgt-todo initial-id="AAMkAGVmMDEzMTM4LTZmYWUtNDdkNC1hMDZiLTU1OGY5OTZhYmY4OAAuAAAAAAAiQ8W967B7TKBjgx9rVEURAQAiIsqMbYjsT5e-T7KzowPTAAAAAAESAAA="></mgt-todo>
```